### PR TITLE
doc: remove extra space

### PR DESCRIPTION
**Pull request by 7h3-3mp7y-m4n**

### CODE REVIEW

File: src/xdocs/config_naming.xml [hunk 1]
```diff
@@ -2098,7 +2098,7 @@ class MyClass {
         <source>
 class MyClass {
   public MyClass() {} // OK
-  public void MyClass() {} // violation,  method Name 'MyClass' must not
+  public void MyClass() {} // violation, method Name 'MyClass' must not
```
Comment (romani) [inline src/xdocs/config_naming.xml:2101]: Do same update in MethodNameCheck.java

**Review by romani — CHANGES_REQUESTED** [review]
Items

File: src/xdocs/config_naming.xml (hunk 1 above)
Comment (romani) [reply src/xdocs/config_naming.xml:2101]: https://github.com/checkstyle/checkstyle/blob/d2648e425bfbd9dfd8f1915a565591e6dba646aa/src/main/java/com/puppycrawl/tools/checkstyle/checks/naming/MethodNameCheck.java#L146

Comment (nrmancuso) [reply src/xdocs/config_naming.xml:2101]: @7h3-3mp7y-m4n please always allow reviewer to resolve items.

**Comment by romani** [comment]
squash all commits in one. There shoudl be one commit in PR

**Comment by 7h3-3mp7y-m4n** [comment]
ok

**Comment by 7h3-3mp7y-m4n** [comment]
I have only edited documentation. why am I facing such issues which are non relevant to my work

**Comment by romani** [comment]
Please read CI failures, they tells what you should do.

Make commit message single line.

**Pull request closed without merging**